From c197389c9d8ccdbfcfe1dc31da6b4ce69b90ecf6 Mon Sep 17 00:00:00 2001
From: Wenzong Fan <wenzong.fan@windriver.com>
Date: Thu, 23 Oct 2014 03:43:27 -0400
Subject: [PATCH] Configure: fix LSOF_CCV assignment

Allow to use specified LSOF_CC to get the version of linux compiler.

Upstream-Status: Pending

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
---
 Configure |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Configure b/Configure
index 29c1292..e5a199f 100755
--- a/Configure
+++ b/Configure
@@ -2705,8 +2705,8 @@ LOCKF_OWNER4
     if test "X$LSOF_CC" = "X"	# {
     then
       LSOF_CC=cc
-      LSOF_CCV=`$LSOF_CC -v 2>&1 | sed -n 's/.*version \(.*\)/\1/p'`
     fi	# }
+    LSOF_CCV=`$LSOF_CC -v 2>&1 | sed -n 's/.*version \(.*\)/\1/p'`
     if test "X$LINUX_CONF_CC" = "X"	# {
     then
       LINUX_CONF_CC=$LSOF_CC
-- 
1.7.9.5

